src/clean-data.js

```diff
@@ -151,12 +151,13 @@ export const cleanAttributes = (attributes, currentSchema, schemas) => {
  * @param {Object} ctx
  * @returns {Object}
  */
-export const cleanData = ({ id, attributes }, ctx) => {
+export const cleanData = ({ id, attributes, ...rest }, ctx) => {
```

Comment:
Is this because the /files endpoint doesn't have the v4 response format?

Reply:
Yes and similarly for the user content type ....